guide/16-geoenrichment-guides/part5_generate_reports.ipynb

```diff
@@ -0,0 +1,928 @@
+{
```

Comment:
<p>Some explanations to "Summary Reports" and "Maps" will be better since they dont look like usu. categories such as demographics...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>